interfaces/builtin/uio.go

```diff
@@ -112,6 +128,14 @@ func (iface *uioInterface) UDevConnectedPlug(spec *udev.Specification, plug *int
 	return nil
 }
 
+func (iface *uioInterface) getUIOName(input string) string {
```

Comment:
As commented above, this method is not needed

Reply:
done